src/main/java/it/pagopa/interop/probing/eservice/operations/util/OffsetLimitPageable.java

```diff
@@ -0,0 +1,60 @@
+package it.pagopa.interop.probing.eservice.operations.util;
+
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+
+import lombok.Builder;
+
+@Builder
+public class OffsetLimitPageable implements Pageable {
+
+	private Integer limit;
+	private Integer offset;
+	private Sort sort;
+
+	@Override
+	public int getPageNumber() {
+		return limit;
+	}
+
+	@Override
+	public int getPageSize() {
+		return limit;
+	}
+
+	@Override
+	public long getOffset() {
+		return offset;
+	}
+
+	@Override
+	public Sort getSort() {
+		return sort;
+	}
+
+	@Override
+	public Pageable next() {
+		return null;
+	}
+
+	@Override
+	public Pageable previousOrFirst() {
+		return null;
+	}
+
+	@Override
+	public Pageable first() {
+		return null;
+	}
+
+	@Override
+	public Pageable withPage(int pageNumber) {
+		return null;
+	}
+
+	@Override
+	public boolean hasPrevious() {
+		return false;
+	}
```

Comment:
as we have adapted to the front-end component, which you provided, which only uses offsets, limits.
e.g. first page called with offset 0 , limit 10
       second page called with offset 10 , limit 10
so we have not implemented them because they are unused

Reply:
If it's not too much effort, I would suggest to implement them to avoid unexpected behaviours on future changes